packages/multipart_form/multipart_form.0.1.0/opam

```diff
@@ -30,6 +30,7 @@ depends: [
   "fmt"
   "ke"         {>= "0.4"}
   "alcotest"   {with-test}
+  "bigstringaf" {>= "0.2.0" & < "0.9.0"}
```

Comment:
```suggestion
  "bigarray-compat"
```
```
#=== ERROR while compiling multipart_form.0.1.0 ===============================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.14.0 | file:///home/opam/opam-repository
# path                 ~/.opam/4.14/.opam-switch/build/multipart_form.0.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p multipart_form -j 31
# exit-code            1
# env-file             ~/.opam/log/multipart_form-1806-da82d4.env
# output-file          ~/.opam/log/multipart_form-1806-da82d4.out
### output ###
# File "lib/dune", line 4, characters 12-27:
# 4 |  (libraries bigarray-compat bigstringaf ke stdlib-shims unstrctrd unstrctrd.parser base64 base64.rfc2045
#                 ^^^^^^^^^^^^^^^
# Error: Library "bigarray-compat" not found.
# -> required by library "multipart_form" in _build/default/lib
# -> required by _build/default/META.multipart_form
# -> required by _build/install/default/lib/multipart_form/META
# -> required by _build/default/multipart_form.install
# -> required by alias install
```

Reply:
that's what I had earlier as well, but there's a lacking bigstringaf dependency as well (take a look at `bigarray-compat bigstringaf`). And now bigstringaf needs to be constrained (0.1 does not provide all used bindings, 0.9 will not be compatible in respect to type (since `implicit_transitive_deps false`).

Reply:
ok then. I’ve put it back and will merge as soon as linter is green